infrastructure/main.tf

```diff
@@ -1,4 +1,6 @@
 terraform {
+  required_version = "0.13"
```

Comment:
```suggestion
  required_version = "1.3.7"
```

Reply:
🤦🏼

Reply:
```bash
vscode ➜ /workspaces/FlowEHR (t-young31/scaffolding) $ terraform --version
Terraform v1.3.6
on linux_arm64
```
can we pin to 1.3 ?

Reply:
Just updated the dc to 1.3.7, but yeah we can pin more generically